QRCoder/BitmapByteQRCode.cs

```diff
@@ -97,9 +98,12 @@ public byte[] GetGraphic(int pixelsPerModule, byte[] darkColorRgb, byte[] lightC
                 }
             }
 
-            //finalize with terminator
-            bmp.AddRange(new byte[] { 0x00, 0x00 });
```

Comment:
Terminator removed?

Reply:
Yeah i removed it, I haven't found any information that a terminator is necessary in the bitmap file. The necessary zeros that it is 4 byte aligned are already added in line 90 to 96: `if (sideLength%4 != 0) ...`

Reply:
Just double-checked once again with the same QRcode.bmp saved from Paint, the result: the file is the same except two bytes are different: `the horizontal and vertical resolution of the image. (pixel per metre, signed integer)`. But no terminator bytes are created by other apps which create .bmp files.

Reply:
> Just double-checked once again with the same QRcode.bmp saved from Paint, the result: the file is the same except two bytes are different: `the horizontal and vertical resolution of the image. (pixel per metre, signed integer)`. That depends on the use case. But no terminator bytes are created by other apps which create .bmp files.

Perfect!

Reply:
For the life of me, I can't remember why I wrote the terminator. I certainly didn't make it up, but I can't find any source that mentions this type of terminator. So as far as I'm concerned, it can go.